app/models/school.rb

```diff
@@ -7,5 +7,5 @@ class School < ApplicationRecord
 
   validates :responsable_contact,
     presence: true,
-    length: { minimum: 9, maximum: 10 }
+    length: { minimum: 9 }
```

Comment:
This does not belong in this PR :)

Reply:
I know! 😞 By mistake I did it in the master branch and now is in all the new branches...
I had too change it because FFaker was creating phone numbers longer than 10 digits..